src/tokenize_sentence/tests.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 vtext developers
+//
+// Licensed under the Apache License, Version 2.0,
+// <http://apache.org/licenses/LICENSE-2.0>. This file may not be copied,
+// modified, or distributed except according to those terms.
+
+use crate::tokenize::Tokenizer;
+use crate::tokenize_sentence::{UnicodeSentenceTokenizer, UnicodeSentenceTokenizerParams};
+
+#[test]
+fn test_unicode_sentence_tokenizer() {
+    let s = "Here is one. Here is another! This trailing text is one more";
+
+    let tokenizer = UnicodeSentenceTokenizerParams::default().build().unwrap();
+    let tokens: Vec<&str> = tokenizer.tokenize(s).collect();
+    let b: &[_] = &[
+        "Here is one. ",
+        "Here is another! ",
```

Comment:
It's interesting that it doesn't strip spaces. I guess it shouldn't matter later processing of sentences and this makes it reversible.

Reply:
Yes - its usefully for reversibility but doesn't align with other packages that usually remove spaces.